rbac/internal/views.py

```diff
@@ -127,3 +139,24 @@ def migration_progress(request):
 
         return HttpResponse(json.dumps(payload), content_type="application/json")
     return HttpResponse(f'Method "{request.method}" not allowed.', status=405)
+
+
+def run_seeds(request):
+    """View method for running seeds.
+
+    POST /_private/api/seeds/run/?seed_types=permissions,rolese,groups
+    """
+    if request.method == "POST":
+        args = {}
+        option_key = "seed_types"
+        valid_values = ["permissions", "roles", "groups"]
+        seed_types_param = request.GET.get(option_key)
+        if seed_types_param:
```

Comment:
May be able to use https://github.com/RedHatInsights/insights-rbac/blob/449c39e0fc9de547165b70a502d362a6784cfa15/rbac/management/utils.py#L155 or something similar here?

Reply:
Good suggestion! Unfortunately I don't think `validate_and_get_key` will work out of the box in this case because a) the default value should actually be empty, in which case it would fail unless I pass in "groups,permissions,roles" as the default value, but also b) because it expects (and returns) one value, vs doing the "optional multiple values" with splitting a string like "groups,roles".